src/components/node/news_story/index.tsx

```diff
@@ -84,12 +86,51 @@ export const NewsStoryFull = ({ node }: NodeNewsStoryProps) => {
   )
 }
 
+/** Teaser news story. */
+export const NewsStoryTeaser = ({ node, headingLevel }: NodeNewsStoryProps) => {
+  if (node?.type !== 'node--news_story') return
+
+  const TitleTag = ({ children, className }) => {
+    const Heading = headingLevel ? headingLevel : 'h2'
+    return <Heading className={className}>{children}</Heading>
+  }
+
+  return (
+    <div className="usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row">
+      <div className="usa-width-two-thirds">
+        <TitleTag className="vads-u-font-size--md medium-screen:vads-u-font-size--lg medium-screen:vads-u-margin-bottom--0p5">
+          <a href={node.path?.alias}>{node.title}</a>
+        </TitleTag>
+        <p className="vads-u-margin-y--0">
+          {truncateWordsOrChar(node.field_intro_text, 60, true)}
+        </p>
+      </div>
+      <div className="usa-width-one-third stories-list vads-u-order--first medium-screen:vads-u-order--initial vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
+        <MediaImageComponent
+          image={node.field_media}
+          imageStyle={'2_1_medium_thumbnail'}
+        />
+      </div>
+    </div>
+  )
```

Comment:
since we're not null checking `node` up higher we should probably add some just to be safe :)
- node?.field_media
- node?.field_intro_text
- node?.path?.alias
- node?.title